dockstore-integration-testing/src/test/java/io/dockstore/common/CommonTestUtilities.java

```diff
@@ -577,6 +584,24 @@ public static <T> T getArbitraryURL(String url, GenericType<T> type, ApiClient c
         return getArbitraryURL(url, type, client, "application/zip");
     }
 
+    /**
+     * This retrieves a resource file using getResourceAsStream, which works for retrieving resource files packaged in a jar.
+     * This allows other repos, like dockstore-support, to use utility methods that require resource files from this package.
+     * @param resourceFileName
+     * @return
+     */
+    public static File getUniversalResourceFile(String resourceFileName) {
+        File tempResourceFile;
+        try (InputStream inputStream = Objects.requireNonNull(CommonTestUtilities.class.getClassLoader().getResourceAsStream(resourceFileName))) {
+            tempResourceFile = File.createTempFile(resourceFileName, null);
```

Comment:
You can add a call to `deleteOnExit` to make sure this file is deleted when the vm exits.